pkg/lint/rules/deprecated_properties_should_have_comment.go

```diff
@@ -0,0 +1,28 @@
+package rules
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/schemalint/pkg/lint"
+	"github.com/giantswarm/schemalint/pkg/lint/utils"
+	"github.com/giantswarm/schemalint/pkg/schemautils"
+)
+
+type DeprecatedPropertiesShouldHaveComment struct{}
+
+func (r DeprecatedPropertiesShouldHaveComment) Verify(schema *schemautils.ExtendedSchema) lint.RuleResults {
+	ruleResults := &lint.RuleResults{}
+	callback := func(schema *schemautils.ExtendedSchema) {
+		if schema.Deprecated && schema.Comment == "" {
+			ruleResults.Add(fmt.Sprintf("Deprecated property '%s' should have a comment.", schema.GetHumanReadableLocation()))
```

Comment:
Should we do this in order to show what key we expect?

```suggestion
			ruleResults.Add(fmt.Sprintf("Deprecated property '%s' should have a $comment.", schema.GetHumanReadableLocation()))
```

Reply:
Yes, good idea.